### DISCUSSION TITLE
Why  I cannot visit my mother country for winter break?

### DISCUSSION THREAD

**Discussion by bambambku** [community/community#86614] [GitHub Education]
### Hi everyone,

Visiting Poland, having a break from studying in the UK, I have been rejected Student Dev PAck although I provided all the documents.

Why I can't visit my parents?

**Comment by juanpflores** [comment]
Hi Bambambku,

I have checked, and your academic status appears to have been verified. Congratulations!

You should already have access to [GitHub Global Campus](https://education.github.com/globalcampus/student), where you can connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more.

You may have already received an email that your academic benefits are available. If not, then stay tuned! Your academic benefits will become available within 72 hours of your verification.

You can always check your academic status [here](https://education.github.com/discount_requests/application).

**Comment by bambambku** [comment]
Thank you very much 😊
